core/modules/views_ui/css/views_ui.admin.theme.css

```diff
@@ -49,7 +49,7 @@
     url("../images/sprites.png"),
     -moz-linear-gradient(
       -90deg,
-      #ffffff 0px,
```

Comment:
Were these changes intentional? (or perhaps automatically changed by an editor?)  I think our standard is to show all 6 characters of a hex code.